hubconf.py

```diff
@@ -0,0 +1,10 @@
+dependencies = ["torch"]
```

Comment:
In this project, single quote is preferred instead of double quote!

```suggestion
dependencies = ['torch']
```